engine/src/osxprinter.cpp

```diff
@@ -447,8 +447,6 @@ bool MCMacOSXPrinter::Reset(MCStringRef p_name, PMPrintSettings p_settings, PMPa
 		t_err = PMGetPageFormatPaper(t_page_format, &t_paper);
 		if (t_err != noErr)
 			t_success = false;
-		else
-			PMRetain(t_paper);
 	}
```

Comment:
I'm not sure this is correct - according to the docs here https://developer.apple.com/documentation/applicationservices/1461319-pmgetpageformatpaper?language=objc the returned PMPaper value should not be released without retaining first.

Reply:
Okay - I'll take another look - it definitely fixed the leak I saw, but perhaps not in the right way :)